lms/templates/courseware/courses.html

```diff
@@ -89,8 +88,8 @@ <h1 id="gymshorts">Gym Shorts</h1>
       <div class="col-md-3 sidebar">
         <%include file='../jobs/find_jobs.html' />
 
-        ## include sidebar content from git repo
-        ${_(urllib2.urlopen('https://gymnasium.github.io/static-site-content/upcoming-courses.html').read())}
+        ## include sidebar content from cms
+        ${gymcms.render('upcoming-courses')}
```

Comment:
This is a cool design pattern! Embedding upcoming courses with a call to a remote URL.

Reply:
@natea — we use these includes throughout the site so that we can make updates to content without making changes to the templates. The content lives in one of two branches in a repo that gets automatically built (jekyll) and deployed separately for staging and/or production.